src/index.tsx

```diff
@@ -7,9 +7,11 @@ import React, { StrictMode } from 'react'
 import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
 import { HashRouter as Router } from 'react-router-dom'
-import { AnalyticsProvider } from '@gooddollar/web3sdk-v2'
+import { AnalyticsProvider, GoodIdContextProvider } from '@gooddollar/web3sdk-v2'
 import { NewsFeedProvider } from '@gooddollar/web3sdk-v2'
 import { PostHogProvider } from 'posthog-react-native'
+import { Provider as PaperProvider } from 'react-native-paper'
```

Comment:
since when do we us react-native-paper?

Reply:
I cannot remember the specifics, should have written them down.
but the native-base modal with React.fragment was not working properly on android devices

I moved it to paper with portal in this commit: 
https://github.com/GoodDollar/GoodWeb3-Mono/pull/128/commits/d33b05328e3a268e9b2cfb6166afb3fc0cc45886

Reply:
for modal to work you need to have the provider at the top. I doubt that native-base modals arent android compatible

Reply:
@sirpy so you want me to refactor it back to nativebase modals?

the paper modals work fine across all platforms

Reply:
ok open an issue with link to the commit